components/bookshelf/discover/SearchBox.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState, useContext } from "react"
+import {
+  SearchButton,
+  SearchForm,
+  SearchInput,
+  SearchLabel,
+} from "../../styled/bookshelf"
+import { BookContent } from "../../styled/bookshelf"
+import { BooksContext } from "./../../../context/BooksContext"
```

Comment:
You probably want to try using another pathing method. I'm not sure if we still need other configurations, but can you try this out?

`"context/BooksContext"`